include/crypto/asn1err.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 2020-2020 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
This is strange!! Looks like the updated err scripts are doing this??

Reply:
Anyway the state from the PR is OK.